tools/create_freq_index.cpp

```diff
@@ -52,8 +52,7 @@ void dump_index_specific_stats(pisa::pefopt_index const& coll, std::string const
         }
     }
 
-    pisa::stats_line()("type", type)("eps1", conf.eps1)("eps2", conf.eps2)(
-        "fix_cost", conf.fix_cost)("docs_avg_part", long_postings / docs_partitions)(
+    pisa::stats_line()("type", type)("docs_avg_part", long_postings / docs_partitions)(
```

Comment:
So what is the idea here? Are we dropping configuring these params (because they just work this way) or are planning to bring it back with command-line option?

Reply:
The idea is that there is no need to print these numbers. They are not configurable, so I personally don’t really need to see them every time. 
If we will make it configurable through CLI then we will print them somewhere